serial.c

```diff
@@ -2,17 +2,37 @@
 #include <dos.h>
 #include "serial.h"
 
-#ifdef _MSC_VER
+#if defined( __DJGPP__ )
+#include <sys/farptr.h>
+#include <go32.h>
+#include <dpmi.h>
+#define Interrupt
+#define Far
+#define Farpeekw(s, o)              _farpeekw((s),(o))
+#define _Outp(a,b)                  (outp(a,b),(b))
+#define _Outpw(a,w)                 (outpw(a,w),(w))

```

Comment:
On Tue, 22 Dec 2015, Karl Stenerud wrote:
> > @@ -2,17 +2,37 @@
> >  #include <dos.h>
> >  #include "serial.h"
> >  
> > -#ifdef _MSC_VER
> > +#if defined( __DJGPP__ )
> > +#include <sys/farptr.h>
> > +#include <go32.h>
> > +#include <dpmi.h>
> > +#define Interrupt
> > +#define Far
> > +#define Farpeekw(s, o)              _farpeekw((s),(o))
> > +#define _Outp(a,b)                  (outp(a,b),(b))
> > +#define _Outpw(a,w)                 (outpw(a,w),(w))
> Can you change these identifiers to either start with a lowercase letter or not start with an underscore? Oddly enough, this style of identifier is actually reserved.
> 7.1.3 Reserved identifiers: All identifiers that begin with an underscore and either an uppercase letter or another
> underscore are always reserved for any use.

Done and thanks for the hint.

best regards,
Przemek